server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/StandardAction.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.acl.authorization;
+
+import io.vertx.ext.auth.authorization.Authorization;
+import io.vertx.ext.auth.authorization.PermissionBasedAuthorization;
+import io.vertx.ext.auth.authorization.impl.PermissionBasedAuthorizationImpl;
+import org.apache.cassandra.sidecar.exceptions.ConfigurationException;
+
+import static org.apache.cassandra.sidecar.acl.authorization.WildcardAction.WILDCARD_TOKEN;
+
+/**
+ * Standard actions need an exact match between allowed actions.
+ */
+public class StandardAction implements Action
+{
+    protected final String name;
+
+    public StandardAction(String name)
+    {
+        if (name == null || name.isEmpty())
```

Comment:
Should we also prevent the WILDCARD_TOKEN to be present on the name?

Reply:
Makes sense. But with `StandardAction` we can only create a `PermissionBasedAuthorization` that doesn't understand matching wildcard permissions. It does an exact String match. So I think we are good to avoid the check here. Wdyt?

Reply:
What worries me is relying on external callers for valid standard actions. Regardless of what is being passed, or who passes it, if a StandardAction should not have wildcards, I think this should make the check.

Reply:
@bbotella but that would violate, having only class specific details in each class right? because we will have `WILDCARD_TOKEN` in `StandardAction` class

Reply:
I think each class, should check for its required conditions and not worry about other conditions wdyt?